backend/plugin/db/sasl.go

```diff
@@ -4,17 +4,31 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"runtime"
 	"strings"
 	"sync"
 
 	"github.com/pkg/errors"
 )
 
+func init() {
+	if err := os.Setenv("KRB5_CONFIG", DftKrbConfPath); err != nil {
```

Comment:
Instead of setting global environment, let's set it when we start the sub-processes.

Reply:
It looks a bit crude, but unfortunately, there are two places where the KRB5_CONFIG is needed...
- The 'kinit' command in the subprocess.
- gohive.Connect() in the same process.

Reply:
Let's add a comment.